### PR TITLE
deps(monorepo): bump yarn to latest v4

### PR DESCRIPTION
**Pull request by clemsos**
# Description

This PR moves the monorepo to yarn v4, with shiney terminal colors and improved features (read [blog post](https://yarnpkg.com/blog/release/4.0)) -- fix #12925

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by clemsos** [comment]
Still need to check if the approach to versioning used in CI still works

**Comment by clemsos** [comment]
```
1581 packages were added to the project, and 3762 were removed (- 348.78 MiB).
```

👀

File: governance/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@unlock-protocol/governance",
-  "packageManager": "yarn@3.5.1",
+  "packageManager": "yarn@4.0.1",
```
Comment (julien51) [inline governance/package.json:3]: Do we want to put that at the root of our repo as well?

Comment (julien51) [reply governance/package.json:3]: Oh it's there I think, so any reason why it needs to be here too?

File: provider/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "provider",
-  "packageManager": "yarn@3.5.1",
+  "packageManager": "yarn@4.0.1",
```
Comment (julien51) [inline provider/package.json:3]: same ?

**Review by julien51 — APPROVED** [review]
exciting!

**Comment by clemsos** [comment]
> Still need to check if the approach to versioning used in CI still work. Basically need to check all tasks at root level work correctly

ok all these seem to work properly

File: governance/package.json (hunk 1 above)
Comment (clemsos) [reply governance/package.json]: let's delete them

**Review by searchableguy — APPROVED** [review]

**Pull request merged**